SMLHelper/Handlers/CraftTreeHandler.cs

```diff
@@ -86,45 +45,8 @@ public static ModCraftTreeRoot GetExistingTree(CraftTree.Type Scheme)
         /// <seealso cref="ModCraftTreeLinkingNode.AddCraftingNode(TechType)"/>
         public static void AddCraftingNode(CraftTree.Type craftTree, TechType craftingItem, params string[] stepsToTab)
         {
-            ModCraftTreeRoot root = GetExistingTree(craftTree);
-
-            if (root == null)
-                return;
-
-            ModCraftTreeTab tab = root.GetTabNode(stepsToTab);
-
-            if (tab == null)
-                return;
-
-            tab.AddCraftingNode(craftingItem);
-        }
-
-        /// <summary>
-        /// Adds a collection of new crafting nodes to the root of the specified crafting tree, at the provided tab location.
-        /// </summary>
-        /// <param name="craftTree">The target craft tree to edit.</param>
-        /// <param name="craftingItems">The item to craft.</param>
-        /// <param name="stepsToTab">
-        /// <para>The steps to the target tab.</para>
-        /// <para>These must match the id value of the CraftNode in the crafting tree you're targeting.</para>
-        /// <para>Do not include "root" in this path.</para>
-        /// </param>
-        /// <seealso cref="GetExistingTree"/>
-        /// <seealso cref="ModCraftTreeRoot.GetTabNode(string[])"/>
-        /// <seealso cref="ModCraftTreeLinkingNode.AddCraftingNode(IEnumerable{TechType})"/>
-        public static void AddCraftingNode(CraftTree.Type craftTree, IEnumerable<TechType> craftingItems, params string[] stepsToTab)
-        {            
-            ModCraftTreeRoot root = GetExistingTree(craftTree);
-
-            if (root == null)
-                return;
-
-            ModCraftTreeTab tab = root.GetTabNode(stepsToTab);
-
-            if (tab == null)
-                return;
-
-            tab.AddCraftingNode(craftingItems);
+            // Add to game
```

Comment:
Just about all the methods that used to use the `GetExistingTree` method still have this line in their XML Documentation
` /// <seealso cref="GetExistingTree"/>`
Remove these entries to avoid any confusion in the documentation, if this hasn't already caused compiler warnings.

Reply:
Removed :)

Reply:
Verified